internal/exec/stages/files/files.go

```diff
@@ -83,16 +93,18 @@ func (s stage) Run(config types.Config) error {
 		return fmt.Errorf("failed to create units: %v", err)
 	}
 
-	if err := s.createCrypttabEntries(config); err != nil {
-		return fmt.Errorf("creating crypttab entries: %v", err)
-	}
+	if !s.isLive {
```

Comment:
For each action we're skipping in the live case, can we add a comment explaining why it's correct to do so?